lib/mongoid/traversable.rb

```diff
@@ -93,8 +113,10 @@ def self.add_discriminator_mapping(value, klass=self)
       #
       # @api private
       def self.get_discriminator_mapping(value)
-        self.discriminator_mapping[value]
+        self.discriminator_mapping[value] if self.discriminator_mapping
```

Comment:
The docstring needs to be updated to indicate that nil is a possible return value, and when this would happen.

Reply:
👍